.circleci/config.yml

```diff
@@ -962,7 +962,7 @@ workflows:
               ignore:
                 - main
                 - release-please--branches--main--components--kurtosis
-                - gh-readonly-queue/main/.*
+                - /gh-readonly-queue.*/
```

Comment:
Interesting, why does this disable the checker in the queue? Are branches not named `/gh-readonly-queue/main/` something?

Reply:
Haven’t tried I thought this regular expression would cover that case, the
wrapping // are just regular expression notation

The branches are indeed called

gh-readonly-queue/main/pr-hash